ledger/participant-state/kvutils/tools/engine-benchmark/src/benchmark/scala/ledger/participant/state/kvutils/tools/engine/benchmark/Replay.scala

```diff
@@ -37,11 +37,15 @@ import scala.collection.JavaConverters._
 final case class TxEntry(
     tx: SubmittedTransaction,
     participantId: ParticipantId,
-    submitter: Ref.Party,
+    submitters: List[Ref.Party],
     ledgerTime: Time.Timestamp,
     submissionTime: Time.Timestamp,
     submissionSeed: crypto.Hash,
-)
+) {
+  def singleSubmitterOrThrow() =
+    if (submitters.length == 1) submitters.head
+    else sys.error("Multi-party submissions are not supported")
```

Comment:
```suggestion
    if (submitters.length == 1)
      submitters.head
    else
      sys.error("Multi-party submissions are not supported")
```

Reply:
The existing code was formatted using `./fmt`. Do you know if it's possible to change the scalafmt config to use the above style?

Reply:
I'm not sure, we'll need to look into it -- I agree this should be automated if possible.

Reply:
Add curly braces, and it'll handle it. :slightly_smiling_face: